packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaFormActions.tsx

```diff
@@ -0,0 +1,194 @@
+}
+
+export function TableColumnSchemaFormActions(
+  props: TableColumnSchemaFormActionsProps,
+) {
+  const { disabled = false, annotationColumnModelsQuery } = props
+  const dispatch = useSetAtom(tableColumnSchemaFormDataAtom)
+
+  const columnModels = useAtomValue(tableColumnSchemaFormDataAtom)
+  const allSelected = getIsAllSelected(columnModels)
+  const numSelected = getNumberOfSelectedItems(columnModels)
+
+  const [
+    showUseRecommendedSizesConfirmationDialog,
+    setShowUseRecommendedSizesConfirmationDialog,
+  ] = useState(false)
+
+  const showUseRecommendedSizesButton =
+    annotationColumnModelsQuery.isLoading || annotationColumnModelsQuery.data
+  const numberOfColumnsBelowRecommendedSizes = annotationColumnModelsQuery.data
+    ? getNumberOfColumnModelsWithValuesBelowMinimumRecommendedSizes(
+        columnModels,
+        annotationColumnModelsQuery.data,
+      )
+    : 0
+
+  return (
+    <Box display={'flex'} alignItems={'flex-start'} gap={1}>
+      <Button
+        aria-label={'Select All'}
+        variant={'outlined'}
+        color={'neutral'}
+        onClick={() => {
+          dispatch({ type: 'toggleSelectAll' })
+        }}
+        sx={{ alignSelf: 'stretch' }}
+        disabled={disabled || columnModels.length == 0}
+      >
+        <Checkbox
+          sx={{ mr: 1 }}
+          checked={allSelected}
+          indeterminate={numSelected > 0 && !allSelected}
+          disabled={disabled || columnModels.length == 0}
+        />
+        <Typography variant="smallText1" color={'text.secondary'}>
+          {numSelected} selected
+        </Typography>
+      </Button>
+      <ButtonGroup sx={{ alignSelf: 'stretch' }}>
+        <Button
+          aria-label={'Move Down'}
+          variant={'outlined'}
+          color={'neutral'}
+          onClick={() => {
+            dispatch({ type: 'moveDown' })
+          }}
+          disabled={disabled || numSelected == 0}
+        >
+          <South fontSize={'small'} />
+        </Button>
+        <Button
+          aria-label={'Move Up'}
+          variant={'outlined'}
+          color={'neutral'}
+          onClick={() => {
+            dispatch({ type: 'moveUp' })
+          }}
+          disabled={disabled || numSelected == 0}
+        >
+          <North fontSize={'small'} />
+        </Button>
+      </ButtonGroup>
+      <Button
+        aria-label={'Delete'}
+        variant={'outlined'}
+        color={'neutral'}
+        onClick={() => {
+          dispatch({ type: 'delete' })
+        }}
+        sx={{ alignSelf: 'stretch' }}
+        disabled={disabled || numSelected == 0}
+      >
+        <IconSvg fontSize={'small'} icon={'delete'} wrap={false} />
+      </Button>
+      <Box flexGrow={1}>{/* spacer */}</Box>
+      <Link
+        sx={{ display: 'block', py: '10px', mr: '10px', textAlign: 'right' }}
+        href="https://sagebionetworks.jira.com/wiki/spaces/DRAFT/pages/1972404304/Organizing+Data+With+Tables"
```

Comment:
I think this should point to the published help docs static site page instead.
https://help.synapse.org/docs/Organizing-Data-With-Tables.2011038095.html ?

Reply:
Definitely, thanks for catching that

Reply:
```suggestion
        href="https://help.synapse.org/docs/Organizing-Data-With-Tables.2011038095.html"
```